resources/assets/lib/components/user-card-brick.tsx

```diff
@@ -70,7 +78,17 @@ export default class UserCardBrick extends React.Component<Props> {
         <div className='user-card-brick__username u-ellipsis-overflow'>
           {this.props.user.username}
         </div>
+        {this.props.onRemoveClick != null && (
+          <div className='user-card-brick__remove'>
+            <span className='fas fa-times' onClick={this.handleRemoveClick}/>
```

Comment:
onClick on the div? Or the hover effect not be part of `div` which has extra padding (which highlights but doesn't actually remove)